src/client/components/Editor.tsx

```diff
@@ -32,6 +34,37 @@ class Editor extends React.PureComponent<EditorProps> {
         }
     }
 
+  componentDidMount(): void {
+      // TODO: why does using gutterclick yield weird highlighting?
+      this.aceComponent.current.editor.on("gutterclick", function(e) {
+        var target = e.domEvent.target;
+
+        if (target.className.indexOf("ace_gutter-cell") == -1){
```

Comment:
Using `classList` is more robust:

`if (!target.classList.contains('ace_gutter-cell')) {`